lib/utils.pm

```diff
@@ -1269,6 +1270,23 @@ sub run_scripted_command_slow {
     clear_console;
 }
 
+=head2 sle15_workaround_broken_patterns
+
+    sle15_workaround_broken_patterns();
+
+Workaround to continue installation even with error pop-ups about missing patterns.
+
+=cut
+
+sub sle15_workaround_broken_patterns {
```

Comment:
Please avoid moving more functions into utils.pm. More than 1k lines is too much already. Better use a base class for installation.